src/main/scala/org/camunda/feel/impl/interpreter/BuiltinFunctions.scala

```diff
@@ -1150,6 +1153,15 @@ object BuiltinFunctions extends FunctionProvider {
       }
     )
 
+  private def isDefinedFunction =
```

Comment:
(Optional) maybe replace it with "is undefined"

Reply:
I thought about it. I prefer `is defined(x)` because it looks more intuitive when inverting it with `not(is defined(x))`. Instead of having a double negation `not(is undefined(x))`.